index.js

```diff
@@ -197,10 +198,22 @@ client.on('message', async (message) => {
   else if (isSayCommand(message.content)) {
     const languageCode = 'en';
     const text = isSayCommand(message.content);
-    const api = `https://translate.google.com.vn/translate_tts?ie=UTF-8&q=${text}+&tl=${languageCode}&client=tw-ob`;
+
+    const curl = `curl 'https://translate.google.com/translate_tts?ie=UTF-8&q=${text}&tl=${languageCode}&client=tw-ob'`;
+    curl += " -H 'Referer: http://translate.google.com/' -H 'User-Agent: stagefright/1.2 (Linux;Android 5.0)' > tts.mp3";
+    // const api = `https://translate.google.com.vn/translate_tts?ie=UTF-8&q=${text}+&tl=${languageCode}&client=tw-ob`;
+
+    exec(curl, (error, stdout, stderr) => {

```

Comment:
?????

Reply:
DONT WORRY
IT IS NOW FIXED
AXIOS GET() IS USED